dev/scripts/load_test_customer_data.sh

```diff
@@ -422,7 +422,7 @@ build_all(){
 }
 
 # ---execute---
-provider_arg=`echo ${1} |tr [a-z] [A-Z]`
+provider_arg=`echo ${1^^} |tr [a-z] [A-Z]`
 
 case ${provider_arg} in
```

Comment:
The `tr` command is uppercasing the input string. If you want to use this technique, it should be done in the case statement.
```suggestion
case ${provider_arg^^} in
```